crates/topos/src/options.rs

```diff
@@ -29,11 +34,29 @@ pub(crate) struct Opt {
     )]
     pub(crate) verbose: u8,
 
+    /// Home directory for the configuration
+    #[arg(
+    long,
+    env = "TOPOS_HOME",
+    default_value = get_default_home(),
+    global = true
+    )]
+    pub(crate) home: PathBuf,
+
     #[command(subcommand)]
     pub(crate) commands: ToposCommand,
 }
 
-#[derive(Subcommand, Debug)]
+/// If no path is given for the --home argument, we use the default one
+/// ~/.config/topos for a UNIX subsystem
+fn get_default_home() -> OsString {
```

Comment:
This is currently generating problems with the snapshot tests. Locally, the `OsString` will add the `USERNAME` to the path, whereas in CI, this will result in something different.

Thinking about how we can adjust the tests for this...

Reply:
You can use `regex` to parse the result output in the test and replace the path with a static one such as `home`